README.md

```diff
@@ -74,6 +74,61 @@ perfplot log_Array1k_19-09-2019_12-13-49
 
 This will generate a file called `log_Array1k_19-09-2019_12-13-49.pdf`.
 
+# Save results to SQL database
+
+## Requirements
+***ODB 2.5.0 (this is required for gcc 7.4.0):***
+Follow the instructions [here](https://www.codesynthesis.com/products/odb/doc/install-build2.xhtml)
+ to install build2 toolchain, odb compiler and runtime libraries.
+
+Remember to install the correct gcc plugin, for example (for gcc 7):
+```
+sudo apt-get update
+sudo apt-get install gcc-7-plugin-dev
+```
+Additionally, if you want to use MYSQL database, you will need to build a not yet released version,
+so instead of executing:
+```
+bpkg build libodb-mysql
+bpkg install libodb-mysq
+
+```
+you will need to execute:
+```
+bpkg build libodb-mysql@https://git.codesynthesis.com/odb/libodb-mysql.git#fix-bind-decl
+bpkg install libodb-mysq
```

Comment:
@monidzik I think there is a typo here. `l` is missing in `bpkg install libodb-mysq`

Reply:
Yes, you are right, thanks!

Reply:
Check https://github.com/ApexAI/performance_test/pull/84/files?file-filters%5B%5D=.cpp&file-filters%5B%5D=.hpp&file-filters%5B%5D=.md&file-filters%5B%5D=.txt#diff-04c6e90faac2675aa89e2176d2eec7d8R105  this one should also contain a `l` at the end

Reply:
oh, right, I'll change it